apps/omg/lib/omg/validators/merge_transaction_validator.ex

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.MergeTransactionValidator do
+  @moduledoc """
+  Checks for transaction attributes. Used by OMG.Fees.
```

Comment:
some more detailed description would be nice, since this is now a very specialized module. I can draft one up:

```
Decides whether transactions qualify as "merge" transactions that use a single currency, single recipient address and have fewer outputs than inputs. This decision is necessary to know by the child chain to not require the transaction fees.
```

Feel free to cut or expand this description as you see fit, possibly splitting it across `moduledoc` and `is_merge_transaction?` doc.

Reply:
Thanks, will update.